bigbluebutton-html5/imports/ui/components/actions-bar/component.jsx

```
@@ -40,18 +40,32 @@ class ActionsBar extends PureComponent {
             switch (plugin.type) {
               case ActionsBarItemType.BUTTON:
+                buttonProps = {
+                  key: `${plugin.type}-${plugin.id}`,
+                  onClick: plugin.onClick,
+                  hideLabel: true,
+                  color: 'primary',
+                  size: 'lg',
+                  circle: true,
+                  label: plugin.tooltip,
+                };
+                if ('iconName' in plugin?.icon) {
```

Biome (1.9.4)

[error] Unsafe usage of optional chaining.

If it short-circuits with 'undefined' the evaluation will throw TypeError here:


(lint/correctness/noUnsafeOptionalChaining)